script.js

```diff
@@ -1,4 +1,284 @@
+var diceObject = []; // This stores the list of dice values of all players
```

Comment:
It's more intuitive to use plurals for arrays. If not, diceObject sounds like a single object.